lib/iris/etc/pp_save_rules.txt

```diff
@@ -761,14 +761,14 @@ THEN
 
 #MDI
 IF
-    cm.fill_value is not None
+    isinstance(cm.data, ma.core.MaskedArray)
```

Comment:
@lbdreyer Replace this with `ma.isMaskedArray(cm.data)` ? But this forces the loading of the data.

Slightly confusing is that for both of these PP save rule, in v1.13, there should be no change here. I don't quite understand that (at the moment) as there is no `fill_value` attribute on a `cm` (cube) ... so how did these save rule work in v1.13 ...